plugins/action/zos_unarchive.py

```diff
@@ -120,4 +124,13 @@ def run(self, tmp=None, task_vars=None):
                 )
             else:
                 result.update(dict(failed=True))
+
+        if not module_args.get("remote_src", False) and uss_format:
+            self._remote_cleanup(tmp_files)
+
         return result
+
+    def _remote_cleanup(self, tempfile_path):
+        """Removes the temporary file in a managed node created for a local
```

Comment:
Not a blocking comment, but the comment "Removes the temporary file in a managed node created for a local script" came from the script module :) , probably should read slightly differently , eg "Removes the temporary file in a managed node created to copy supported archives on the managed node"

Reply:
Going to resolve this as is non-blocking